src/main/kotlin/org/opensearch/observability/index/ObservabilityIndex.kt

```diff
@@ -54,6 +54,7 @@ internal object ObservabilityIndex {
     private const val NOTEBOOKS_INDEX_NAME = ".opensearch-notebooks"
     private const val OBSERVABILITY_MAPPING_FILE_NAME = "observability-mapping.yml"
     private const val OBSERVABILITY_SETTINGS_FILE_NAME = "observability-settings.yml"
+    private const val SEARCH_ACTION_PREFERENCE = "_primary"
```

Comment:
Would `_primary_first` be a better option? https://opensearch.org/docs/latest/api-reference/search/#the-preference-query-parameter